README.md

```diff
@@ -245,17 +247,18 @@ A QueueMessage can be created in one of two ways:
     ```php
     $messageFactory = $container->get(\LinioPay\Idle\Message\MessageFactory::class);
     
-    /** @var QueueMessage $message */
-    $message = $messageFactory->createMessage([ // Create a QueueMessage configured to work with SQS (Because it was defined in the config)
+    /** @var SendableMessage $message */
+    $message = $messageFactory->createSendableMessage([ // Create a QueueMessage configured to work with SQS (Because it was defined in the config)
```

Comment:
SQS? Here it's not tied to AWS SQS, right?

Reply:
In this example yeah, its intended to go to SQS because 'my-queue' (queue_identifier) is configured to use SQS.  Check the sample configuration earlier in the guide:

```
                'types' => [ // Define the queues where the QueueMessages are coming from
                    'my-queue' => [		                    'my-queue' => [
                        'parameters' => [		                        'parameters' => [
                            // Inherit SQS as its service		                            // Inherit SQS as its service
                        ],		                        ],
                    ],		                    ],
                    'my-task-queue' => [		                    'my-task-queue' => [
                        'parameters' => [		                        'parameters' => [
                            'service' => GoogleCloudTasks::IDENTIFIER, // Override the service to use Google CloudTasks instead of AWS SQS		                            'service' => GoogleCloudTasks::IDENTIFIER, // Override the service to use Google CloudTasks instead of AWS SQS
                        ]		                        ]
                    ]		                    ]
                ]		                ]```

Reply:
I think it is a followup from here:
https://github.com/LinioPay/idle/pull/133/files#diff-04c6e90faac2675aa89e2176d2eec7d8L173